heat/core/tests/test_devices.py

```diff
@@ -2,62 +2,89 @@
 import os
 import heat as ht
 
-if os.environ.get("DEVICE") == "gpu" and ht.torch.cuda.is_available():
+envar = os.getenv("HEAT_USE_DEVICE", "cpu")
+
+if envar == "cpu":
```

Comment:
This code repeats in several places. Would it make sense to have a heat unittest class that derives from the standard unittest and would encapsulate the code in the setupClass method()?